### DISCUSSION TITLE
ShadcnUI components throwing error while compiling.

### DISCUSSION THREAD

**Discussion by 0xabinash** [shadcn-ui/ui#3835] [Q&A]
I'm getting the error for all the Shadcn components that are used in my project, the components work just fine during the development phase, but throws error while building the app for deployment.

Error: src/components/ui/toast.jsx (1:0): Error when using sourcemap for reporting an error: Can't resolve original location of error.

project repo: [repo](https://github.com/0xabinash/demo)

**Comment by yap-qliro** [comment]
I have the same issue

```
src/components/ui/shadcn/dialog.tsx (1:0): Error when using sourcemap for reporting an error: Can't resolve original location of error.
src/components/ui/shadcn/label.tsx (1:0): Error when using sourcemap for reporting an error: Can't resolve original location of error.

**Reply by yap-qliro** [reply]
It seems `"use client";` is the issue, removing `"use client";` solves it.
https://github.com/vitejs/vite/issues/15012
